requirements.txt

```diff
@@ -17,3 +17,4 @@ pynvml == 11.5.0
 triton >= 2.1.0
 outlines == 0.0.34
 tiktoken == 0.6.0 # Required for DBRX tokenizer
+# sparsetensors # Commented out, as pypi package not yet available - create from source.
```

Comment:
You can put GitHub repos in here like
```
sparsetensors @ git+https://github.com/neuralmagic/sparsetensors@main
```

However, we may want to put this in the setup.py in the sparse extra deps

Reply:
I think this would break wheel builds and definitely pypi pushes